src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Attributes.jl

```diff
@@ -46,7 +46,7 @@ Affine scheme morphism
   from [x1, x2, x3]  scheme(x1)
   to   [x1, x2, x3]  affine 3-space over QQ
 given by the pullback function
-  x1 -> 0
+  x1 -> x1
```

Comment:
Why do these doctests change? There is no change in the code above that would explain it

Reply:
We don't reduce the generators anymore when there's a "variable-to-variable" map. Mathematically all these are the same.